sparse/tests/test_coo.py

```diff
@@ -471,6 +471,14 @@ def test_trinary_broadcasting_pathological(shapes, func, value, fraction):
     assert_eq(fs, func(*dense_args), equal_nan=True)
 
 
+def test_almost_zero_float():
+    x = np.array([1e-20, 1e-15, 1e-14], dtype=np.float16)
```

Comment:
As above, it'd be nice to add tests of different ranges of inputs. Inputs smaller than the "rest of the array" should be cut off.